src/js/control-bar/progress-control/play-progress-bar.js

```diff
@@ -14,6 +15,20 @@ import './time-tooltip';
  */
 class PlayProgressBar extends Component {
 
+  /**
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The {@link Player} that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   */
+  constructor(player, options) {
+    super(player, options);
+    this.update = Fn.throttle(Fn.bind(this, this.update), 25);
```

Comment:
in the seek bar, we throttle to 30ms: https://github.com/videojs/video.js/blob/master/src/js/control-bar/progress-control/seek-bar.js#L25,  probably worth using the same thing everywhere. Maybe even have this value as a property exported from Fn utils?

Reply:
Yeah let's do that.